### ISSUE TITLE
[Question] Why did you create 5 workers

### ISSUE THREAD

**Issue by AhtashamImtiyaz** [jeromewu/tesseract.js-video#1]
Hi dude,

I like your example. But i didn't get why did you create 5 workers in the loop and added them in the schedule? Is there any specific reason?

**Comment by ekatioz** [comment]
Hey @AhtashamImtiyaz,
I think he did this, because he is queuing up recognition-jobs with every interval tick.
More workers can improve the performance when there are more jobs in the queue. As described here: https://github.com/naptha/tesseract.js/blob/master/docs/api.md#createscheduler-scheduler

Correct me if I'm wrong. 😄

**Comment by AhtashamImtiyaz** [comment]
Yes you are right. I have tested myself as well. We have to create multiple workers to improve performance
